experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -240,6 +255,10 @@ for i in 1:(n_stem + n_leaf)
 end
 
 Y.canopy.energy.T = drivers.TA.values[1 + Int(round(t0 / DATA_DT))] # Get atmos temperature at t0
+
+Y.snow.S .= 0.0
```

Comment:
I'm thinking about how we can clean up the user experience in drivers, and I think it would be nice to have a `set_initial_values` function where we have default values for each prognostic + diagnostic variable, with the option for the user to pass in alternative values that overwrite the defaults. What do you think?

Again, not for this PR, just an idea :)

Reply:
I think that's a good idea, maybe for the new `src/simulation` folder (which will eventually replace `ClimaLandSimulations`) - we should have a design meeting at some point, I think maybe in January. 
I think right now the priorities are snow model and calibrations, as well as spatially varying params and stable simulations... once all this is done, we should think of the simulations redesign? (happy to discuss)

Reply:
I agree! This might tie into @Sbozzolo's recent addition of setting IC from a checkpoint file. It might be that global runs start from a setup similar to what we have by default, whereas fluxnet sites set IC based on the data. but either way we need to run for a while to spinup